stock_reserve/__openerp__.py

```diff
@@ -32,6 +32,7 @@
              ],
  'demo': [],
  'data': ['view/stock_reserve.xml',
+          'view/stock_location.xml',
```

Comment:
8.0.2.0.0, for 2 reasons:

* Having the full version nomenclature.
* This new version implies a migration script.